installation/sql/postgresql/base.sql

```diff
@@ -113,16 +113,17 @@ INSERT INTO "#__assets" ("id", "parent_id", "lft", "rgt", "level", "name", "titl
 (87, 18, 124, 125, 2, 'com_modules.module.97', 'Recently Added Articles', '{}'),
 (88, 18, 126, 127, 2, 'com_modules.module.98', 'Logged-in Users', '{}'),
 (89, 18, 128, 129, 2, 'com_modules.module.90', 'Login Support', '{}'),
-(90, 1, 165, 166, 1, 'com_scheduler', 'com_scheduler', '{}'),
-(91, 1, 167, 168, 1, 'com_associations', 'com_associations', '{}'),
-(92, 1, 169, 170, 1, 'com_categories', 'com_categories', '{}'),
-(93, 1, 171, 172, 1, 'com_fields', 'com_fields', '{}'),
-(94, 1, 173, 174, 1, 'com_workflow', 'com_workflow', '{}'),
-(95, 1, 175, 176, 1, 'com_guidedtours', 'com_guidedtours', '{}'),
+(90, 1, 167, 168, 1, 'com_scheduler', 'com_scheduler', '{}'),
+(91, 1, 169, 170, 1, 'com_associations', 'com_associations', '{}'),
+(92, 1, 171, 172, 1, 'com_categories', 'com_categories', '{}'),
+(93, 1, 173, 174, 1, 'com_fields', 'com_fields', '{}'),
+(94, 1, 175, 176, 1, 'com_workflow', 'com_workflow', '{}'),
+(95, 1, 177, 178, 1, 'com_guidedtours', 'com_guidedtours', '{}'),
 (96, 18, 130, 131, 2, 'com_modules.module.109', 'Guided Tours', '{}'),
-(97, 1, 177, 178, 1, 'com_scheduler.task.1', 'com_scheduler.task.1', '{}'),
-(98, 1, 179, 180, 1, 'com_scheduler.task.2', 'com_scheduler.task.2', '{}'),
-(99, 1, 181, 182, 1, 'com_scheduler.task.3', 'com_scheduler.task.3', '{}');
+(97, 1, 179, 180, 1, 'com_scheduler.task.1', 'com_scheduler.task.1', '{}'),
+(98, 1, 181, 182, 1, 'com_scheduler.task.2', 'com_scheduler.task.2', '{}'),
+(99, 1, 183, 184, 1, 'com_scheduler.task.3', 'com_scheduler.task.3', '{}'),
+(100, 18, 132, 133, 2, 'com_modules.module.110', 'Joomla! Community and News', '{}');
 
 SELECT setval('#__assets_id_seq', 100, false);
```

Comment:
```suggestion
SELECT setval('#__assets_id_seq', 101, false);
```
On PostgreSQL when adding a new record to a table with an auto-incremented ID column and specifying the ID in the INSERT statement like here, you also have to increment the next value of the corresponding sequence.

Reply:
Thanks good to know!